starboard/common/storage.cc

```diff
@@ -17,24 +17,35 @@
 namespace starboard {
 
 StorageRecord::StorageRecord()
-    : user_(SbUserGetCurrent()), record_(kSbStorageInvalidRecord) {
+    :
+#if SB_API_VERSION < 16
+      user_(SbUserGetCurrent()),
+#endif
+      record_(kSbStorageInvalidRecord) {
   Initialize();
 }
 
-StorageRecord::StorageRecord(SbUser user)
-    : user_(user), record_(kSbStorageInvalidRecord) {
+StorageRecord::StorageRecord(const char* name)
+    :
+#if SB_API_VERSION < 16
+      user_(SbUserGetCurrent()),
+#endif
+      name_(name),
+      record_(kSbStorageInvalidRecord) {
   Initialize();
 }
 
-StorageRecord::StorageRecord(const char* name)
-    : user_(SbUserGetCurrent()), name_(name), record_(kSbStorageInvalidRecord) {
+#if SB_API_VERSION < 16
+StorageRecord::StorageRecord(SbUser user)
+    : user_(user), record_(kSbStorageInvalidRecord) {
   Initialize();
 }
 
 StorageRecord::StorageRecord(SbUser user, const char* name)
     : user_(user), name_(name), record_(kSbStorageInvalidRecord) {
   Initialize();
 }
+#endif
```

Comment:
Tiny nitpick: we generally add a comment after endif, like `#endif // SB_API_VERSION < 16` - makes the long ifdef blocks easier to grok. That's here and elsewhere below.